install_sct

```diff
@@ -382,6 +382,7 @@ if [ "${SCT_DIR}" != "${SCT_SOURCE}" ]; then
   mkdir -p ${SCT_DIR}/install
   cp -vRp ${SCT_SOURCE}/install ${SCT_DIR}
   cp -vRp ${SCT_SOURCE}/testing ${SCT_DIR}/.
+  mkdir -p ${SCT_DIR}/documentation/imgs && cp -vRp ${SCT_SOURCE}/documentation/imgs/* ${SCT_DIR}/documentation/imgs/*
```

Comment:
actually, i tested it and it does not work: it copies "*" instead of the correct file (i'm on OSX):
~~~
mkdir -p ${SCT_DIR}/documentation/imgs && cp -vRp ${SCT_SOURCE}/documentation/imgs/* ${SCT_DIR}/documentation/imgs/*
/Users/julien/code/sct//documentation/imgs/logo_sct.png -> /Users/julien/testsct/documentation/imgs/*
~~~

The following works:
~~~
mkdir -p ${SCT_DIR}/documentation/imgs && cp -vRp ${SCT_SOURCE}/documentation/imgs/*.* ${SCT_DIR}/documentation/imgs/
/Users/julien/code/sct//documentation/imgs/logo_sct.png -> /Users/julien/testsct/documentation/imgs/logo_sct.png
~~~

Reply:
@jcohenadad  Hmmm, strange, worked on my laptop (also macOS), what does exactly gets copied? what do you mean by "*" being copied?

Reply:
@jcohenadad yes! my mistake ... corrected!

Reply:
hmmm ... still I can see there is a "*.*" copied

```
drwxr-xr-x  2 amir  staff     64 15 Jun 00:44 *.*
drwxr-xr-x  4 amir  staff    128 15 Jun 00:44 .
drwxr-xr-x  3 amir  staff     96 15 Jun 00:44 ..
-rw-r--r--  1 amir  staff  79795 14 Jun 23:49 logo_sct.png
```

Reply:
This should be fine 
```
mkdir -p ${SCT_DIR}/documentation/imgs && cp -vRp ${SCT_SOURCE}/documentation/imgs/* ${SCT_DIR}/documentation/imgs
```

Reply:
yes i confirm that works